### PR TITLE
Database completed

### PR DESCRIPTION
**Pull request by jasonrees1**
Please take a look at my database

### CODE REVIEW

**Review by harrycoulton — COMMENTED** [review]
fine

**Comment by harrycoulton** [comment]
looks fine to me mate

File: netflix_series_2019-09-23.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+# ------------------------------------------------------------
+
+CREATE TABLE `favourites` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `TITLE` varchar(50) NOT NULL DEFAULT '',
+  `START DATE` date NOT NULL,
+  `END DATE` date DEFAULT NULL,
+  `NUMBER OF SEASONS` int(3) NOT NULL,
+  `NUMBER OF EPISODES` int(3) NOT NULL,
+  `DIRECTOR` varchar(30) NOT NULL DEFAULT '',
```
Comment (sjr-mayden) [inline netflix_series_2019-09-23.sql]: Is 30 going to be enough?

Comment (sjr-mayden) [reply netflix_series_2019-09-23.sql]: I would increase this and title to be varchar 255 in case you need it in the future

**Review by sjr-mayden — CHANGES_REQUESTED** [review]

**Review by sjr-mayden — APPROVED** [review]

**Pull request merged**